<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TwoFADialog</class>
 <widget class="QWidget" name="TwoFADialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>994</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1031</width>
     <height>1001</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Users/benrh/OneDrive/Pictures/im3.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>330</y>
     <width>361</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 196, 133);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="labelStatus1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 700 9pt &quot;Segoe Print&quot;;</string>
    </property>
    <property name="text">
     <string>Enter the 2FA code</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCode">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(53, 74, 140);
font: 700 10pt &quot;Segoe Print&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-color: rgb(53, 74, 140);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>6</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSubmit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 700 9pt &quot;Segoe Script&quot;;</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatus2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Invalid or expired code</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
